sdk/tables/azure-data-tables/azure/data/tables/_error.py

```diff
@@ -215,6 +205,31 @@ def _process_table_error(storage_error, table_name=None):
     _reraise_error(decoded_error)
 
 
+def _reprocess_error(decoded_error, identifiers=None):
+    error_code = decoded_error.error_code
+    message = decoded_error.message
+    invalid_input = "The number of keys specified in the URI does not match number of key properties for the resource"
+    invalid_query_parameter_value = "Value for one of the query parameters specified in the request URI is invalid"
+    properties_need_value = "The values are not specified for all properties in the entity"
+    table_does_not_exist = "The table specified does not exist"
+    if (error_code == "InvalidInput" and invalid_input in message or # pylint: disable=too-many-boolean-expressions
+        error_code == "InvalidQueryParameterValue" and invalid_query_parameter_value in message or
+        error_code == "PropertiesNeedValue" and properties_need_value in message or
+        error_code =="TableNotFound" and table_does_not_exist in message
+        ):
+        args_list = list(decoded_error.args)
+        args_list[0] += "\nA possible cause of this error could be that the account URL used to"\
+            "create the Client includes an invalid path, for example the table name. Please check your account URL."
+        decoded_error.args = tuple(args_list)
+
+    if (identifiers is not None and error_code == "InvalidXmlDocument" and len(identifiers) > 5):
+        raise ValueError(
+            "Too many access policies provided. The server does not support setting more than 5 access policies"\
+                "on a single resource."
+            )
+    raise decoded_error
```

Comment:
What happens if we just use `raise` instead of `raise decoded_error`?

Reply:
This is a good point! I don't see difference actually, so changed to `raise`.

Reply:
Update: it will occur an error with pylint
```
************* Module azure.data.tables._error
azure/data/tables/_error.py:239: [E0704(misplaced-bare-raise), _reprocess_error] The raise statement is not inside an except clause
```
I'll revert the change back.

Reply:
Have changed to raise after the place that invokes this function.